### DISCUSSION TITLE
Working with multiple sites

### DISCUSSION THREAD

**Discussion by markdav-is** [oqtane/oqtane.framework#758] [Q&A]
We have two main sites. Let's call them "Admin" and "Public". I've been working in Admin and now I want to add Public.   When I look at the Site Management in Oqtane I see one site at localhost:44357 and I'm not sure how to add a site that would not be localhost. Is there some guidance on making a 2nd site in the same tenant? 

![image](https://user-images.githubusercontent.com/311063/94197575-d57dc780-fe6a-11ea-8e10-56e809bfae23.png)

**Comment by sbwalker** [comment]
@markdav-is Did you click on the Add Site button? On the modal UI there is a field for Aliases. By default it displays the current domain ( for convenience ) but this is where you would specify the custom domain for this site ( ie. www.domain.com ). You can also use subfolders as well ie. localhost:44357/public for microsites.

I am not really sure why you would create totally separate sites for Admin and Public - as Oqtane is designed to support both personas in the same site simply by using the role based security model?

**Reply by markdav-is** [reply]
Thanks for the notes, I hadn't thought of using a sub-folder.  That might be the best solution.  We have two sets of customers, folks on the web that are registering for events, think eventbrite. The login there is either Oqtane, or a SAML service from another site where they are already a member.  Different theme with no menus, more of a shopping-cart.  If Oqtane has good enough performance, we'll expose that on a public URL and use that for the registration site.  We'll need to scale that out with load balancers we can get 30K hits in a few hours.  The admin site is completely internal for staff use.  This feels like two different sites to me, and we've created two distinctly different themes.  We need to test Oqtane for performance, but would like to use it for the public facing site as well.

**Reply by mikecasas** [reply]
@markdav-is Is your "eventbrite" module public?

**Reply by markdav-is** [reply]
Not a public repo. It's our enterprise app.
